mmv1/products/securitycenter/ProjectSccBigQueryExports.yaml

```
@@ -0,0 +1,130 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ProjectSccBigQueryExports'
```

Drive-by comment: These should be singular, not plural. The file name will also need to change.

```suggestion
name: 'ProjectSccBigQueryExport'
```

---

Addressed.